.github/workflows/label-first-time-contributor.yml

```diff
@@ -0,0 +1,38 @@
+name: Label First-Time Contributor PRs
+
+on:
+  pull_request_target:
+    types:
+      - opened
+
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Add 'new contributor' label
+        uses: actions/github-script@v6.3.3
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            // Get a list of all issues created by the PR opener
+            const creator = context.payload.sender.login
+            const opts = github.rest.issues.listForRepo.endpoint.merge({
+              ...context.issue,
+              creator,
+              state: 'all'
+            })
+
+            const labelToAdd = 'new contributor';
+            const issues = await github.paginate(opts)
+
+            for (const issue of issues) {
+              if (issue.number === context.issue.number) {
```

Comment:
Unless I missed something, this iterates through a list of all issues created by the PR opener and adds a label to the PR if it is in the list, which is equivalent to just adding the label to every created PR.

Reply:
I honestly don't know what to say; it seems to work here
https://github.com/ankidroid/Anki-Android/blob/89441d7936cee653d61fbd42881d2e5be2ce361e/.github/workflows/label.yml#L154-L184
https://github.com/ankidroid/Anki-Android/pull/14118 the action runs here and does not label it as from a new contributor
https://github.com/ankidroid/Anki-Android/actions/runs/5489823104/jobs/10004466562?pr=14118#step:2:4